src/renderer/components/configure-recording-set-page/index.tsx

```diff
@@ -1,268 +1,66 @@
 import log from 'electron-log';
-import React, { FC, Fragment, MouseEventHandler, useContext, useEffect, useState } from 'react';
-import Col from 'react-bootstrap/Col';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import { useTranslation } from 'react-i18next';
-import CSSTransition from 'react-transition-group/CSSTransition';
-import useLocalStorage from 'react-use/lib/useLocalStorage';
+import React, { FC, MouseEventHandler, useState } from 'react';
 
-import { RecordingProjectContext } from '../../contexts';
-import { PROJECT_CONFIG_FILENAME } from '../../env-and-consts';
-import { Consumer, RecordingProject, RecordingSet, ScaleKey, SupportedOctave } from '../../types';
-import {
-  checkFileExistence,
-  deleteFolder,
-  ensureFolderExists,
-  getLSKey,
-  join,
-  naiveSerialize,
-  readFile,
-  writeFile,
-} from '../../utils';
-import BackButton from '../back-button';
-import NextButton from '../next-button';
-
-import AddRecordingSetButton from './add-recording-set-button';
-import CreatedRecordingSetList from './created-recording-set-list';
-import SetMetaConfiguration from './set-meta-configuration';
-import SetRecordingListConfiguration from './set-recording-list-configuration';
-import { BuiltInRecordingList } from './types';
+import { Consumer, RecordingSet } from '../../types';
 
+import ConfigureRecordingSet from './configure-recording-set';
 import './show-details.scss';
+import Dvcfg from './dvcfg';
+import ListPreview from './list-preview';
+import OtoIni from './oto-ini';
 
-interface ProjectFile extends RecordingProject {
-  name: string;
-  recordingSets: RecordingSet[];
-}
-
-const DUMMY_PROJECT_FILE = {
-  name: '',
-  rootPath: '',
-  recordingSets: [],
-};
-
-const BUILT_IN_RECORDING_LISTS: BuiltInRecordingList[] = [
-  'デルタ式英語リストver5 (Delta English Ver. 5)',
-  'Z式CVVC-Normal (Z Chinese CVVC - Normal)',
-];
+export type RecordingPageState = 'home' | 'list-preview' | 'oto-ini' | 'dvcfg' | 'external';
```

Comment:
Add a JsDoc describing what `RecordingPageState` is and what these values are. Especially, what is `external`?

Reply:
added doc above